### DISCUSSION TITLE
Renovate doesn't update deps on my public repo

### DISCUSSION THREAD

**Discussion by evis** [renovatebot/renovate#17032] [Request Help]
Repo: https://github.com/evis/scalafix-maven-plugin

I've added renovate app and merged https://github.com/evis/scalafix-maven-plugin/pull/24.

Afterwards app created https://github.com/evis/scalafix-maven-plugin/issues/33. But it didn't create neither PR nor commit to master branch. Why?

**Comment by rarkins** [comment]
You've configured branch automerge, which means Renovate will:
- create the branch, no PR
- wait for branch tests to pass
- automerge directly to master branch

The reasons it has not merged to master yet can include:
- It does not have permissions, or
- It has not run again on your repo since the tests passed

You can:
- Log into app.renovatebot.com to check logs and when it last ran, and
- Tick the `Check this box to trigger a request for Renovate to run again on this repository` box to force it to run

I also recommend that you reduce the number of concurrent branches you have, because every time one of them is merged to master then the others will need rebasing in order to be up to date before merging
